tests/image/classification/test_data.py

```diff
@@ -569,3 +574,41 @@ def test_from_bad_csv_no_image(bad_csv_no_image):
             num_workers=0,
         )
         _ = next(iter(img_data.train_dataloader()))
+
+
+@pytest.mark.skipif(
+    not (_IMAGE_TESTING and _ALBUMENTATIONS_AVAILABLE), reason="image libraries + albumentation aren't installed."
```

Comment:
Should just check `_IMAGE_AVAILABLE` here as I think albumentations will only be installed for the image_extras testing

Reply:
albumentations isn't a dependency. This is just for demo purpose.

Reply:
Seems weird to me to have a test that we don't expect to run